pkg/controllers/cassandra/service/resource.go

```diff
@@ -11,25 +11,28 @@ import (
 func ServiceForCluster(
 	cluster *v1alpha1.CassandraCluster,
 ) *apiv1.Service {
-	svc := apiv1.Service{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:            util.ResourceBaseName(cluster),
-			Namespace:       cluster.Namespace,
-			Labels:          util.ClusterLabels(cluster),
-			Annotations:     make(map[string]string),
-			OwnerReferences: []metav1.OwnerReference{util.NewControllerRef(cluster)},
-		},
-		Spec: apiv1.ServiceSpec{
-			Type: apiv1.ServiceTypeClusterIP,
-			Ports: []apiv1.ServicePort{
-				{
-					Name:       "transport",
-					Port:       int32(9042),
-					TargetPort: intstr.FromInt(9042),
-				},
-			},
-			Selector: util.NodePoolLabels(cluster, ""),
+	return UpdateServiceForCluster(cluster, &apiv1.Service{})
+}
+
+func UpdateServiceForCluster(
```

Comment:
Do we export the functions in resources.go in other packages?

Reply:
I export `ServiceForCluster` so that it can be used in unit tests.